support-frontend/assets/components/thankYou/thankYouModuleData.tsx

```diff
@@ -53,7 +53,6 @@ export const getThankYouModuleData = (
 	csrf: CsrfState,
 	email: string,
 	isOneOff: boolean,
-	amountIsAboveThreshold: boolean,
 	campaignCode?: string,
 ): Record<ThankYouModuleType, ThankYouModuleData> => {
```

Comment:
Is this method used [here](https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/pages/digital-subscriber-thank-you/digitalSubscriptionThankYou.tsx#L48-L56) too?

Reply:
Its should of been removed from [here](https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/pages/digital-subscriber-thank-you/digitalSubscriptionThankYou.tsx#L48-L56) in the merge and fix above